apps/language_server/lib/language_server/providers/formatting.ex

```diff
@@ -71,7 +71,7 @@ defmodule ElixirLS.LanguageServer.Providers.Formatting do
 
   defp advance_pos({line, col}, str) do
     Enum.reduce(String.split(str, "", trim: true), {line, col}, fn char, {line, col} ->
-      if char == "\n" do
+      if char in ["\n", '\r'] do
```

Comment:
Shouldn't this be `"\r"`?

```suggestion
      if char in ["\n", "\r"] do
```